File: 1-easy/problems/1480-running-sum-of-1d-array/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        for i in range(len(nums)-1):
            nums[i+1] += nums[i]
        return nums

File: 1-easy/problems/1480-running-sum-of-1d-array/solution.ts
function runningSum(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: List[int]
  """
  for (i in Array.from({length: (nums}, (_, i) => i).length-1)) {
      nums[i+1] += nums[i]
  return nums;
}

export { runningSum };
